src/components/VoteScreen.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Title from './Title';
+import VoteRow from './VoteRow';
+import axios from 'axios';
+const Wrapper = styled.div`
+  height: 100vh;
+`;
+
+export default function VoteScreen() {
+  const [candidates, setCandidates] = useState(null);
+  const getData = async () => {
+    try {
+      const result = await axios.get(
+        'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates'
+      );
+      result.data = result.data.sort((a, b) => {
+        return b.voteCount - a.voteCount;
+      });
+      setCandidates(result.data);
```

Comment:
1. `axios`로 받아온 `data`를 비구조화할당을 이용해서 이렇게 바로 뽑을 수 있어요!

2. `sort`함수는 복사본을 반환하는게 아니라 원본배열이 정렬되어서 이렇게 할 필요가 없어요~ [참고](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)

3. 화살표함수에서 바로 return할땐 중괄호와 return문을 생략할 수 있습니다~


```suggestion
      const {data}= await axios.get(
        'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates'
      );
      data.sort((a, b) => b.voteCount - a.voteCount);
      setCandidates(data);
```

Reply:
이런 좋은 방법이...!!